source/index.html.md

```diff
@@ -699,6 +700,23 @@ object][ApplePay-PaymentToken] for more information.
   </dt>
   <dd>
     Expiry year of card to charge.
+  </dd>
+    <dt>mobilepayonline[cryptogram]
+    <span class="type">[:base64:]{19,21}</span>
```

Comment:
A tAV is usually 28 chars after Base64. Do you have a reference for this?

Reply:
No. I was high (apparently). I have used 28 during my testing also, and the code requires 28.